test-app-async/src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagenameasync/integration/HmppsQueueResourceTest.kt

```diff
@@ -200,7 +201,7 @@ class HmppsQueueResourceTest : IntegrationTestBase() {
         .jsonPath("messagesFoundCount").isEqualTo(3)
         .jsonPath("messagesReturnedCount").isEqualTo(3)
         .jsonPath("messages..body.MessageId").value(
-          Matchers.contains(
+          containsInAnyOrder(
```

Comment:
with the async nature now of the sending the message were appearing in a different order